dev-tools/package_test.go

```diff
@@ -0,0 +1,289 @@
+package dev_tools
+
+// This file contains tests that can be run on the generated packages.
+// To run these tests use `go test package_test.go`.
+
+import (
+	"archive/tar"
+	"archive/zip"
+	"bytes"
+	"compress/gzip"
+	"flag"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/blakesmith/ar"
```

Comment:
Could we add a vendor directory to the dev-tools directory? I would prefer if we don't mix vendor packages for tools with actual libbeat / beats dependencies. This makes it easier to track where a dependency belongs to. 

Long term I'm thinking also to move dependencies which only belong to one beat into the beat if possible.